cdk/domino_cdk/provisioners/eks/eks_iam.py

```diff
@@ -11,7 +12,7 @@ def __init__(
     ) -> None:
         self.scope = scope
 
-    def provision(self, name: str, cluster_name: str, s3_policy: iam.ManagedPolicy, r53_zone_ids: List[str]):
+    def provision(self, stack_name: str, cluster_name: str, r53_zone_ids: List[str], buckets: Dict[str, Bucket]):
```

Comment:
Regardless of whether we prefer `name` or `stack_name`, I don't think we should have it as `stack_name` in just these two files and still `name` everywhere else.

Reply:
I really prefer "stack_name" everywhere. Just "name" is too generic and already caused a confusion in the previous iteration of this PR.

Do you ask to go through the code and replace name to stack_name?

Reply:
It could be in a followup PR if you prefer, but I'd like to keep it consistent across all the provisioners.

Reply:
If it just in provisioners, I'll do this.